foscam/foscam.py

```diff
@@ -15,6 +15,11 @@
     from urllib.parse import urlencode
 import xml.etree.ElementTree as ET
 from threading import Thread
+try:
+    import ssl
+    ssl_enabled=True
+except ImportError:
+    ssl_enabled=False
```

Comment:
Hi, I see this is an old pull request. I'm interested in seeing ssl being added to the lib. what would be a better way to handle the import error?
maybe if this is the only error this could be merged and a we could have a fix commit later?
or is it ok to submit a new pull request if the author of this pull request isn't going to update this one?
thanks